### PR TITLE
[zh]resync feature-gates in CLI

### PR DESCRIPTION
**Pull request by Zhuzhenghao**
This PR resync page `content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md`.

### CODE REVIEW

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -1040,16 +1096,41 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `MixedProtocolLBService`：允许在同一 `LoadBalancer` 类型的 Service 实例中使用不同的协议。
 <!--
 - `MultiCIDRRangeAllocator`: Enables the MultiCIDR range allocator.
-- `NetworkPolicyEndPort`: Enable use of the field `endPort` in NetworkPolicy objects,
-  allowing the selection of a port range instead of a single port.
+- `MultiCIDRServiceAllocator`: Track IP address allocations for Service cluster IPs using IPAddress objects.
 - `NetworkPolicyStatus`: Enable the `status` subresource for NetworkPolicy objects.
+- `NewVolumeManagerReconstruction`: Enable improved discovery of mounted volumes during kubelet
```
Comment (Zhuzhenghao) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md:1101]: Duplicated `NewVolumeManagerReconstruction`.

Is it a mistake? @tengqm

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -654,29 +673,44 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
   [Storage Capacity](/docs/concepts/storage/storage-capacity/).
   Check the [`csi` volume type](/docs/concepts/storage/volumes/#csi) documentation for more details.
+- `CSIVolumeHealth`: Enable support for CSI volume health monitoring on node.
 -->
 - `CSINodeExpandSecret`：允许在 `NodeExpandVolume` CSI 操作期间将 Secret
   身份验证数据传递到 CSI 驱动以供后者使用。
 - `CSIStorageCapacity`：使 CSI 驱动程序可以发布存储容量信息，并使 Kubernetes
   调度程序在调度 Pod 时使用该信息。参见[存储容量](/zh-cn/docs/concepts/storage/storage-capacity/)。
   详情请参见 [`csi` 卷类型](/zh-cn/docs/concepts/storage/volumes/#csi)。
+- `CSIVolumeHealth`：启用对节点上的 CSI volume 运行状况监控的支持。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md:683]:
```suggestion
- `CSIVolumeHealth`：启用对节点上的 CSI 卷运行状况监控的支持。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md [hunk 3]
```diff
@@ -654,29 +673,44 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
+<!--
+- `CloudControllerManagerWebhook`: Enable webhooks in cloud controller manager.
+- `CloudDualStackNodeIPs`: Enables dual-stack `kubelet --node-ip` with external cloud providers.
+  See [Configure IPv4/IPv6 dual-stack](/docs/concepts/services-networking/dual-stack/#configure-ipv4-ipv6-dual-stack)
+  for more details.
+- `ClusterTrustBundle`: Enable ClusterTrustBundle objects and kubelet integration.
+-->
+- `CloudControllerManagerWebhook`：在云控制器管理器中启用 Webhook。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md:691]:
```suggestion
- `CloudControllerManagerWebhook`：启用在云控制器管理器中的 Webhook。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md [hunk 4]
```diff
@@ -654,29 +673,44 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
+- `CloudDualStackNodeIPs`：启用与外部云提供商的双协议栈 `kubelet --node-ip`。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md:692]:
```suggestion
- `CloudDualStackNodeIPs`：允许在外部云驱动中通过 `kubelet --node-ip` 设置双协议栈。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md [hunk 5]
```diff
@@ -741,15 +776,23 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
+  See docs on [elastic Indexed Jobs](/docs/concepts/workloads/controllers/job#elastic-indexed-jobs)
+  for more details.
 - `EndpointSliceTerminatingCondition`: Enables EndpointSlice `terminating` and `serving`
    condition fields.
 - `EfficientWatchResumption`: Allows for storage-originated bookmark (progress
   notify) events to be delivered to the users. This is only applied to watch operations.
 -->
 - `DynamicResourceAllocation`：启用对具有自定义参数和独立于 Pod 生命周期的资源的支持。
+- `ElasticIndexedJob`：通过同时改变 `spec.completions` 和 `spec.parallelism`
+  使得 `spec.completions == spec.parallelism` 来扩大或缩小索引作业。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md:794]:
```suggestion
  使得 `spec.completions == spec.parallelism` 来对代索引的 Job 执行扩容或缩容。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md [hunk 6]
```diff
@@ -741,15 +776,23 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
+  有关详细信息，请参阅有关 [elastic Indexed Jobs](/zh-cn/docs/concepts/workloads/controllers/job#elastic-indexed-jobs) 的文档。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md:795]:
```suggestion
  有关详细信息，请参阅有关[弹性的带索引的 Job](/zh-cn/docs/concepts/workloads/controllers/job#elastic-indexed-jobs) 的文档。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md [hunk 7]
```diff
@@ -850,8 +893,10 @@ Each feature gate is designed for enabling/disabling a specific feature:
   资源的 `minReplicas` 设置为 0。
 <!--
 - `IPTablesOwnershipCleanup`: This causes kubelet to no longer create legacy iptables rules.
+- `InPlacePodVerticalScaling`: Enables in-place Pod vertical scaling.
 -->
 - `IPTablesOwnershipCleanup`：这使得 kubelet 不再创建传统的 iptables 规则。
+- `InPlacePodVerticalScaling`：启用就地 Pod 垂直缩放。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md:899]:
```suggestion
- `InPlacePodVerticalScaling`：启用就地 Pod 垂直扩缩。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (tengqm) [reply content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md:1101]: Duplication is an error for sure.

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md [hunk 8]
```diff
@@ -1040,16 +1096,41 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
 <!--
 - `MultiCIDRRangeAllocator`: Enables the MultiCIDR range allocator.
-- `NetworkPolicyEndPort`: Enable use of the field `endPort` in NetworkPolicy objects,
-  allowing the selection of a port range instead of a single port.
+- `MultiCIDRServiceAllocator`: Track IP address allocations for Service cluster IPs using IPAddress objects.
 - `NetworkPolicyStatus`: Enable the `status` subresource for NetworkPolicy objects.
+- `NewVolumeManagerReconstruction`: Enable improved discovery of mounted volumes during kubelet
+  startup.
+-->
+- `MultiCIDRRangeAllocator`：启用 MultiCIDR 范围分配器。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md:1104]:
```suggestion
- `MultiCIDRRangeAllocator`：启用 MultiCIDR 网段分配机制。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md [hunk 9]
```diff
@@ -1040,16 +1096,41 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
+- `MultiCIDRServiceAllocator`: 使用 IPAddress 对象跟踪服务集群 IP 的 IP 地址分配。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md:1105]:
```suggestion
- `MultiCIDRServiceAllocator`: 使用 IPAddress 对象跟踪 Service 的集群 IP 的 IP 地址分配。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md [hunk 10]
```diff
@@ -1040,16 +1096,41 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
+  <!-- remove next 2 paragraphs when feature graduates to GA -->
+  <!--
+  Before Kubernetes v1.25, the kubelet used different default behavior for discovering mounted
+  volumes during the kubelet startup. If you disable this feature gate (it's enabled by default), you select
+  the legacy discovery behavior.
+
+  In Kubernetes v1.25 and v1.26, this behavior toggle was part of the `SELinuxMountReadWriteOncePod`
+  feature gate.
+  -->
+  在 Kubernetes v1.25 之前，kubelet 在 kubelet 启动期间使用不同的默认行为来发现挂载的卷。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md:1117]:
```suggestion
  在 Kubernetes v1.25 之前，kubelet 在启动期间使用不同的默认行为来发现挂载的卷。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md [hunk 11]
```diff
@@ -1040,16 +1096,41 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
   [Pod topology spread constraints](/docs/concepts/scheduling-eviction/topology-spread-constraints/)
   when calculating pod topology spread skew.
+- `NodeLogQuery`: Enables querying logs of node services using the `/logs` endpoint.
 -->
-- `MultiCIDRRangeAllocator`：启用 MultiCIDR 范围分配器。
-- `NetworkPolicyEndPort`：在 NetworkPolicy 对象中启用 `endPort` 以允许选择端口范围而不是单个端口。
-- `NetworkPolicyStatus`：为 NetworkPolicy 对象启用 `status` 子资源。
+- `NewVolumeManagerReconstruction`: 在 kubelet 启动期间启用改进的挂载卷的发现。
+  由于这段代码已经进行了重大重构，我们允许在 kubelet 在启动时被卡住或没有从终止的 Pod 上卸载卷的情况下选择退出。
+  请注意，此重构是在 Kubernetes 1.25 中的 `SELinuxMountReadWriteOncePod` alpha 特性门控背后进行的。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md:1133]: 英文版本中有重复项，建议现修订后再次同步。

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md [hunk 12]
```diff
@@ -1198,14 +1281,15 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
-  the StatefulSet controller.
+- `StableLoadBalancerNodeGet`: Enables less load balancer re-configurations by 
+  the service controller (KCCM) as an effect of changing node state.
 - `StatefulSetStartOrdinal`: Allow configuration of the start ordinal in a
   StatefulSet. See
   [Start ordinal](/docs/concepts/workloads/controllers/statefulset/#start-ordinal)
   for more details.
 -->
 - `SizeMemoryBackedVolumes`：允许 kubelet 检查基于内存制备的卷的尺寸约束（目前主要针对 `emptyDir` 卷）。
+- `StableLoadBalancerNodeGet`: 允许由于节点状态的变化，服务控制器（KCCM）可以减少负载均衡器的重新配置。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md:1292]:
```suggestion
- `StableLoadBalancerNodeGet`: 允许服务控制器（KCCM）根据节点状态变化来减少负载均衡器的重新配置。
```

**Pull request merged**